### ISSUE TITLE
Only english language available in plasma regional settings

### ISSUE THREAD

**Issue by ghost** [NixOS/nixpkgs#33987]
## Issue description
Only english language available in plasma regional settings, but all locales generated.

![nixos](https://user-images.githubusercontent.com/29430720/35066325-425c3bfa-fbe1-11e7-9d47-87e9e8eab85b.png)

### Steps to reproduce
1. Open system settings.
2. Open regional settings.

## Technical details
```
- system: `"x86_64-linux"`
 - host os: `Linux 4.9.76, NixOS, 17.09.2754.66b4de79e38 (Hummingbird)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 1.11.16`
```

**Comment by TSleepingCat** [comment]
I think plasma trying find locales in /usr/share, but finds nothing.

**Comment by kevincox** [comment]
This appears it is also a problem with GNOME, presumable these are using the same locale files.

![image](https://user-images.githubusercontent.com/494012/103825725-6a37ba00-5043-11eb-9215-0e1712a7e2c6.png)

**Comment by kevincox** [comment]
I noticed that adding the locales I wanted to `i18n.supportedLocales` solves the issue. However this is documented as being `"all"` which should contain all locals. However setting it explicitly to `"all"` does resolve the issue. So I don't know why the default is being changed.

```
i18n.supportedLocales = ["all"];
```

**Comment by kevincox** [comment]
Oh, I am dumb, I was actually setting it in my config. It was not conflicting because the arrays just got merged as expected.

**Comment by kevincox** [comment]
@florianjacob are you seeing this issue, and are you not setting this value in your config somewhere?

**Comment by florianjacob** [comment]
This was fixed for me with NixOS 19.09, see https://github.com/NixOS/nixpkgs/issues/37741#issuecomment-544139582 - it probably was a bug in plasma packaging that did not make the locale files available to the applications, and this issue can probably be closed. I always had `i18n.defaultLocale = "de_DE.UTF-8";`
